src/pat/contentbrowser/src/ContentBrowser.svelte

```diff
@@ -426,11 +434,15 @@
                                 {#if level.selectable}
                                     <button
                                         class="btn btn-primary btn-xs"
+                                        data-bs-toggle="tooltip"
+                                        data-bs-title="{level.displayPath}"
+                                        data-bs-placement="top"
+                                        data-bs-custom-class="contentbrowser-tooltip"
```

Comment:
wouldn't a `title` attribute be enough for showing a tooltip by browser mechanisms?
We could also augment that later using patternslib or something, if the browser tooltip isn't enough.

Background: I overheard that Twitter Bootstrap is in declining development state. And I get always a bit nervous when we add another dependency to Bootstrap. Also, the browser tooltip mechanism via the `title` attribute is fine enough in most cases.

Reply:
I second to not use Bootstrap here.